Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang AS builder
+
+COPY . /go/src/github.com/rclone/rclone/
+WORKDIR /go/src/github.com/rclone/rclone/
+
+RUN make quicktest
+RUN \
```

Comment:
Can you run `make` instead of `go build -v` does that work?

That will build the correct tag into `rclone version`. You might need CGO_ENABLED=0 too.

That will also strip some debug symbols out of the binary too which make no difference unless you attach a debugger.

Reply:
It works, indeed.
Pushed.